sessionize/lib/src/commonMain/kotlin/co/touchlab/sessionize/platform/NotificationsModel.kt

```diff
@@ -157,11 +94,40 @@ object NotificationsModel {
 
     // Cancel Singular
 
-    private fun cancelReminderNotification(sessionid: Int) {
-        ServiceRegistry.notificationsApi.cancelLocalNotification(sessionid, notificationReminderTag)
-    }
-
     private fun cancelFeedbackNotification() {
         ServiceRegistry.notificationsApi.cancelLocalNotification(feedbackId, notificationFeedbackTag)
     }
+
+    fun recreateNotifications(){
+        backgroundTask({ sessionQueries.mySessions().executeAsList() }) { mySessions ->
```

Comment:
We might want to consider adding `goAsync` to the receiver on the android side since we are going to the background and returning onReceive. This will open up a complication of how to "finish" when we are done. I'll make a new task for it though so we can move forward with this one

Reply:
Sounds good, so I'll leave it for now in this PR.